murdock/main.py

```diff
@@ -79,11 +80,11 @@
     StaticFiles(directory=GLOBAL_CONFIG.work_dir, html=True, check_dir=False),
     name="results",
 )
+murdock.instrumentator.instrument(app).expose(app)
```

Comment:
Why is this moved outside of the startup event function ? Is it because of the way the logger is setup ?

Reply:
I triggered an issue with starlette that it doesn't allow for adding middleware anymore after starting the app.

Reply:
Not sure btw what triggered this. Locally I have starlette-0.24.0

Reply:
Ok, that's not really an issue anyway, let's keep it like this.